### DISCUSSION TITLE
Progress bar component

### DISCUSSION THREAD

**Discussion by hardikpthv** [ing-bank/lion#1692] [Ideas]
Add a progress bar component that basically accepts:
- min, 
- max
- current value 
We can either add attribute supports in [Progress Indicator](https://lion-web.netlify.app/components/progress-indicator/examples/) or create a standalone component - `lion-progress-bar`.

A11Y
- https://www.w3.org/TR/wai-aria-practices-1.2/#range_related_properties
- https://www.magentaa11y.com/checklist-web/progress-bar/

**Comment by gerjanvangeest** [comment]
When we created the Progress Indicator we said to add support for it in this component.

See: https://github.com/ing-bank/lion/pull/857#discussion_r465574254

I still think that makes sense.

**Reply by hardikpthv** [reply]
I have been looking at the `lion-progress-indicator`. It looks like a component with minimal specs. I don't think the progress bar should fit inside as a11y attributes will differ and the behavior of the component is a bit different. Localize part can be used but it would be cleaner to add a separate component itself a.k.a. `lion-progress-bar`.

**Reply by tlouisse** [reply]
The goal would be mainly to make the lion-progress-indicator follow the "[role=progressbar] spec".
That would mean the current aria attributes change, but the purpose of the component remains the same. Indicator serves as a neutral name that is unopinionated about presentation: it can be used for (indeterminate) circular spinners as well, for instance.

Also see: https://lit-and-friends.slack.com/archives/CJGFWJN9J/p1633368533122400?thread_ts=1632902663.101800&cid=CJGFWJN9J

@hardikpthv are you interested in doing a pull request?

**Reply by hardikpthv** [reply]
I already have changes ready as a separate component but if we want to make it part of the `lion-progress-indicator` we need more specs. Right now as @jorenbroekema  mentioned, `lion-progress-indicator` does not do much except the setting `aria-label`. While the progress bar already has more states than just setting A11Y attrs.

![image](https://user-images.githubusercontent.com/9882972/170034025-89d38a87-38e4-4c37-a952-1b9ac4ec1dae.png)

**Reply by jorenbroekema** [reply]
I think this: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/progressbar_role
In combination with what @tlouisse mentioned in Slack:
![image](https://user-images.githubusercontent.com/36734656/173403215-99d8d36c-2e46-4d2a-998c-f274ba1e56a4.png)

is probably a good start in terms of nailing down some specs/failing tests to start with? bit easier to discuss things concretely when we have a start on some (can be very rough) tests.
